src/App.tsx

```diff
@@ -1,109 +1,32 @@
 import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
-
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
-
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
 
-          <p>
-            {' Posted by  '}
+const getUser = (userId: number) => usersFromServer
+  .find(user => user.id === userId)
+  || ({
+    name: 'unknown',
+    email: 'unknown',
+  });
```

Comment:
I guess looks more readable this way
```suggestion
const getUser = (userId: number) => {
  const unknownUser = {
    name: 'unknown',
    email: 'unknown',
  };
  
  return usersFromServer.find(user => user.id === userId) 
    || unknownUser;
}
```